metricbeat/module/system/process/_meta/docs.asciidoc

```diff
@@ -11,6 +11,14 @@ This metricset is available on:
 - Windows
 
 [float]
+=== Cache CmdLine
+
+This metricset caches the command line args for a running process. This means if you alter
+the command line for a process while this metricset is running, these changes are not detected.
+This feature can disabled by adding
+`process.cmdline.cache: false` to the system module configuration.
```

Comment:
I was more thinking out loud. So far I don't think we had any requests for it so I would leave it out. But it is good to have it as a potential option for the future.